Source/bmalloc/libpas/src/libpas/pas_fast_tls.h

```diff
@@ -84,20 +84,17 @@ struct pas_fast_tls {
         } \
     } while (false)
 
-#define PAS_FAST_TLS_GET(static_key, passed_fast_tls) ({ \
```

Comment:
Should we keep the original implementation for the case when __thread is not exist? I'm not sure we should care about that situation, but

Reply:
Yes, we will keep it since it turned out that this is necessary for Darwin.